editor/editor_build_profile.cpp

```diff
@@ -43,13 +43,17 @@
 const char *EditorBuildProfile::build_option_identifiers[BUILD_OPTION_MAX] = {
 	// This maps to SCons build options.
 	"disable_3d",
-	"disable_2d_physics",
-	"disable_3d_physics",
-	"disable_navigation",
+	"disable_physics_2d",
+	"disable_physics_3d",
+	"disable_navigation_2d",
+	"disable_navigation_3d",
 	"disable_xr",
 	"rendering_device", // FIXME: there's no scons option to disable rendering device
```

Comment:
Once this PR is merged, #103100 should be rebased and amended to solve this FIXME, and add the rendering method options here too.

It might also start being a bit heavy UI wise, so maybe rendering features should be moved to their own category, like the text handling features.